semgrep/semgrep/cli.py

```diff
@@ -14,7 +14,6 @@ def cli() -> None:
     """
     To get started quickly, run `semgrep scan --config auto`
     """
-    pass
```

Comment:
This seems to be caused by a change in the pre-commit requirements?

Reply:
Fixed on develop so you can rebase if ever